worlds/pokemon_emerald/options.py

```diff
@@ -540,11 +762,53 @@ class ReceiveItemMessages(Choice):
     option_none = 2
 
 
+class RemoteItems(Toggle):
+    """
+    Instead of placing your own items directly into the ROM, all items are received from the server, including items you find for yourself.
+
+    This enables co-op of a single slot and recovering more items after a lost save file (if you're so unlucky).
+    But it changes pickup behavior slightly and obviously requires connection to the server to receive any items.
+    """
+    display_name = "Remote Items"
+
+
+class RandomizeMusic(Toggle):
+    """
+    Shuffles music played in any situation where it loops. Includes many FRLG tracks.
+    """
+    display_name = "Randomize Music"
+
+
+class RandomizeFanfares(Toggle):
+    """
+    Shuffles fanfares for item pickups, healing at the pokecenter, etc.
+
+    When this option is enabled, pressing B will interrupt most fanfares.
+    """
+    display_name = "Randomize Fanfares"
+
+
+class WonderTrading(DefaultOnToggle):
+    """
+    Allows participation in wonder trading with other players in your current multiworld. Speak with the center receptionist on the second floor of any pokecenter.
+
+    Wonder trading NEVER affects logic.
+
+    Certain aspects of a pokemon species are per-game, not per-pokemon.
+    As a result, some things are not retained during a trade, including type, ability, level up learnset, and so on.
+    Receiving a pokemon this way does not mark it as found in your pokedex.
+    Trade evolutions do not evolve this way; they remain changed to level ups and item use.
```

Comment:
```suggestion
    Trade evolutions do not evolve this way; they remain changed to require level ups and item use.
```
Adding "require" here improves the clarity/readability

Reply:
Reworded the whole phrase in [46f7c86](https://github.com/ArchipelagoMW/Archipelago/pull/2918/commits/46f7c86c42d9c9bb8da81295766fe84e202f2fbf)